milli/src/search/new/ranking_rule_graph/typo/mod.rs

```diff
@@ -0,0 +1,78 @@
+use roaring::RoaringBitmap;
```

Comment:
relevancy changes: 

- a split word is now considered as having 1 typo
- a digram can have split words too: `whit ehorse` (FIXME: consider allowing for split words in trigrams e.g. `white ehor se`)